src/importexport/musicxml/internal/musicxml/exportxml.cpp

```diff
@@ -5396,15 +5397,24 @@ void ExportMusicXml::textLine(TextLineBase const* const tl, staff_idx_t staff, c
 // supported by MusicXML need to be filtered out. Everything not recognized
 // as MusicXML dynamics is written as other-dynamics.
 
+template<typename T>
+inline std::set<QString>& operator<<(std::set<QString>& s, const T& v)
+{
+    s.insert(v);
+    return s;
+}
+
```

Comment:
Why not using the one from containers.h here? Or have this there?

Reply:
Problem with different types
operator in containers.h wants one same type ((std::set<T>& s, const T& v)) 
but here is two different type, one - QString, second - const char*